ocf_data_sampler/numpy_batch/site.py

```diff
@@ -0,0 +1,31 @@
+"""Convert site to Numpy Batch"""
+
+import xarray as xr
+
+
+class SiteBatchKey:
+
+    site = "site"
+    site_capacity_kwp = "site_capacity_kwp"
```

Comment:
can remove site from variables, but keep in string name

Reply:
maybe keep it like that for the moment, and can change with all others

Reply:
Other prefixes have been removed so i think we should remove site here as well